x/stakeibc/keeper/hooks.go

```diff
@@ -35,6 +35,8 @@ func (k Keeper) BeforeEpochStart(ctx sdk.Context, epochInfo epochstypes.EpochInf
 		k.CleanupEpochUnbondingRecords(ctx, epochNumber)
 		// Create an empty unbonding record for this epoch
 		k.CreateEpochUnbondingRecord(ctx, epochNumber)
+		// Transfers in and out of tokens for hostZones which have community pools
+		k.SweepAllCommunityPoolTokens(ctx)
```

Comment:
It was supposed to be the stride epoch and I put it in the wrong place, per discussions yesterday morning we might actually put it in the even faster hourly epoch since there are more steps than in the original plan